### ISSUE TITLE
Question of filter(kernel) size

### ISSUE THREAD

**Issue by mrchaos** [aqibsaeed/Genetic-CNN#11]
Could the size of the filter(kernel) be larger than the size of the feature map?
In case of continue to add layers, it may end up with a larger filter than the size of the feature map.

**Comment by mrchaos** [comment]
See the images in the link for more information.
https://office.fxrobot.kr/sh/X56

**Comment by DarkoNedic** [comment]
You may look [here](https://github.com/aqibsaeed/Genetic-CNN/pull/10/commits/1f7513fa1bd1905b142138020f8c168d1b9d7442) for my type of solution to this.
I am not sure if this is the correct way, but it seems to fix this issue you show in the image.

**Comment by DarkoNedic** [comment]
Try returning this in `apply_convolution()`:

`return tf.nn.relu(tf.add(tf.nn.conv2d(x, weights,[1,1,1,1],padding = "SAME"),biases))`

**Comment by aqibsaeed** [comment]
It can be but it doesn't make intuitive sense to have kernel size larger than number of feature maps.  I would suggest to first study convolutional neural network and then explore architecture search.  Hope this helps.
